schema.json

```diff
@@ -133,6 +133,14 @@
                     "$ref": "#/definitions/Type",
                     "description": "Parameter type"
                 },
+                "ownership": {
+                    "type": "string",
+                    "description": "Ownership of the value",
+                    "enum": [
+                        "with",
+                        "without"
```

Comment:
Some bikeshedding... `with` and `without` feel a bit implicit? What if we used something more explicit like `application` vs `api`?

Reply:
Whoops I shouldn't have enabled automerge. Was thinking of that too, but used "With" and "Without" to match the ownership article. Maybe `with_ownership: boolean`?